source/_UIAHandler.py

```diff
@@ -168,13 +180,25 @@
 	UIA_SystemAlertEventId:"UIA_systemAlert",
 }
 
+localEventHandlerGroupUIAEventIds = {
+	UIA.UIA_SelectionItem_ElementAddedToSelectionEventId,

```

Comment:
I think it is possibly better to keep this global rather than local. Selection tracking is rather like focus tracking in that there usually are not too many events, and they can come from anywhere. In fact the most important cases, such as with search suggestions, and I'm guessing things like emoji pannel:  the selected items have nothing at all to do with the focus.